src/SALib/test_functions/Sobol_G.py

```diff
@@ -81,7 +81,7 @@ def evaluate(values, a=None, delta=None, alpha=None):
         mod_x = shift_of_x - integral
         temp_y = np.abs(2 * mod_x - 1) ** alpha
         y_elements = ((1 + alpha) * temp_y + a) / (1 + a)
-        Y[i] = np.prod(y_elements)
+        Y[i] = y_elements.prod()
```

Comment:
This one is a mistake no?

Reply:
It was intentional as a micro-performance change.

Calling `np.prod` is slightly slower than `array.prod()` due to how Python looks up package functions vs methods.

I realise it's technically out-of-scope for this PR, but I figured it's a one line change... so I might as well do it now while I was at it.